### PR TITLE
 Sabrina Zancotti project week 2 

### PR DESCRIPTION
**Pull request by Zancotti**
Netlify link: https://zancotti-news.netlify.app/

Im proud of the fact that i succeeded with makeing the whole header for all three medias with just adapting the css for them. It took some hours to understand how to do it without haveing to change in the html. It was also the biggest struggle i had codewise. Another struggle has been me being to eager to get into the codeing so i forgot to plan the project. I think I lost some hours from that, but i learned a lot by doing it that way so not regretting it =D

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,380 @@
...
+  }
+  .other-news-text-container {
+    max-width: 300px;
+    margin-top: 10px;
+    height: auto;
+  }
+
+  .item-2,
+  .item-4,
+  .item-6 {
```
Comment (hejmaria) [inline code/style.css:233]: They seem to have the same styling. Maybe could have given them the same name, just to minimize the number of classes?

File: code/index.html [hunk 2]
```diff
@@ -4,19 +4,294 @@
...
+          <img
+            class="big-news-image"
+            src="./big-news-image.jpeg"
+            alt="A photograph of three soldiers walking with guns"
+          />
+        </div>
+        <div class="big-news-text-container">
+          <div class="big-news-text">
+            <a href="#">
+              <h3>
```
Comment (hejmaria) [inline code/index.html:67]: Suggestion: use h1 and h2 before h3, especially for your biggest news. I believe it is more understandable for the search engines.

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,380 @@
...
+  }
+  .header :nth-child(3) {
+    order: 1;
+  }
+  .header :nth-child(4) {
+    order: 3;
+  }
+  .header :nth-child(5) {
+    order: 4;
+  }
```
Comment (hejmaria) [inline code/style.css:184]: Nice to see how you worked this out.

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,380 @@
...
+  .icon-and-text {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    margin-right: 20px;
+  }
+
+  .icon-and-text:hover {
+    font-size: 18px;
+    transition: 1s ease-out;
```
Comment (hejmaria) [inline code/style.css:278]: Is this the animation? Im impressed!

**Review by hejmaria — APPROVED** [review]
The code works perfectly and the page is responsive (mobile/tablet/desktop size). All the requirements were fulfilled, the code was easy to understand and it followed the guidelines.

Impressive navbar with animated icons! Very nice addition with the flipping polaroids (and such nice vacation pics).
I like how you structured the code. Especially the css. It is easy to follow, good job!

File: code/index.html [hunk 5]
```diff
@@ -4,19 +4,294 @@
...
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, minimum-scale=1"
+    />
+    <script
+      src="https://kit.fontawesome.com/7ed1b41d25.js"
+      crossorigin="anonymous"
+    ></script>
```
[the remark this answers is not in the document]
Comment (Zancotti) [reply code/index.html:14]: It is the way to import icons from the website fontawesome =)

File: code/index.html [hunk 6]
```diff
@@ -4,19 +4,294 @@
...
+                    alt="A photograph of a couple on the beach"
+                  />
+                  <div class="polaroid-text">Greetings from Maldives</div>
+                </div>
+              </div>
+              <div class="flip-card-back">
+                <p>Greetings from Maldives</p>
+              </div>
+            </div>
+          </div>
```
[the remark this answers is not in the document]
Comment (Zancotti) [reply code/index.html:258]: Yeah, it looks like a mess but was needed to make the polaroids and then also flip the cards =D

File: code/index.html (hunk 2 above)
Comment (Zancotti) [reply code/index.html:67]: Haha, very true! =) Did miss this =D

File: code/style.css (hunk 1 above)
Comment (Zancotti) [reply code/style.css:233]: Very true! =) changed this!

File: code/style.css (hunk 4 above)
Comment (Zancotti) [reply code/style.css:278]: Yeah it is a part of the animation =) thanks!!

**Comment by Zancotti** [comment]
> The code works perfectly and the page is responsive (mobile/tablet/desktop size). All the requirements were fulfilled, the code was easy to understand and it followed the guidelines.
> 
> Impressive navbar with animated icons! Very nice addition with the flipping polaroids (and such nice vacation pics).
> I like how you structured the code. Especially the css. It is easy to follow, good job!

Thanks Maria, valuable comments and ive edited to follow most of the suggestions. I thought of makeing more space in the html but I actually like they way it is now as I can easily see the flow when the parts are "together"..

Once again, thank you for taking the time to help me with my code and It opened my eyes more to what I can think of for future projects<3

**Comment by Zancotti** [comment]
Unsure if i was supposed to close it or not. So im leaving it open =D

**Pull request closed without merging**